### PR TITLE
Features/contacts-content/106-searchbar

### PR DESCRIPTION
**Pull request by blueretrograde**
-added functionality to the searchbar component
-imported the searchbar component into Contact Content

https://trello.com/c/9VOMGb4Q/106-add-search-functionality-to-contact-list-page

### CODE REVIEW

File: src/components/ContactContent/ContactContent.jsx [hunk 1]
```diff
@@ -3,38 +3,75 @@ import BankContact from "../BankContact/BankContact";
...
+import Searchbar from "../Searchbar/Searchbar";
+import Button from "../../components/Button/Button";
 
 const ContactContent = ({numContacts=4}) => {
     const [allContacts, setAllContacts] = useState([]);
 
+    const [searchText, setSearchText] = useState("");
+    const [userContacts, setUserContacts] = useState();
+
+
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.jsx:15]: have only new line after declaring states

File: src/components/ContactContent/ContactContent.jsx [hunk 2]
```diff
@@ -3,38 +3,75 @@ import BankContact from "../BankContact/BankContact";
 import {useEffect, useState} from "react";
 import { getUserContacts } from "../../utils/apiUtils";
 import creditCard from "../../assets/images/credit-card.png";
+import Searchbar from "../Searchbar/Searchbar";
+import Button from "../../components/Button/Button";
 
 const ContactContent = ({numContacts=4}) => {
     const [allContacts, setAllContacts] = useState([]);
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.jsx:11]: remove blank line

File: src/components/ContactContent/ContactContent.jsx [hunk 3]
```diff
@@ -3,38 +3,75 @@ import BankContact from "../BankContact/BankContact";
...
+                )
+            })
+        )
+    
+        }
+            
     useEffect(() => {
-        getData();
+        getData()
+
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.jsx:42]: remove extra line

File: src/components/ContactContent/ContactContent.jsx [hunk 4]
```diff
@@ -3,38 +3,75 @@ import BankContact from "../BankContact/BankContact";
...
-    })
+    },[searchText])
 
     return (
         <div className="contactList">
             <h3 className="contactList__title">Contact List</h3>
-            <p className="contactList__desc">All your friends and family financial details in one place. Easily transfer currency internationally at the best possible rates.</p>
-            <div className="contactList__headings">
+            <div className="contactList__options">
+                <p className="contactList__desc">All your friends and family financial details in one place. Easily transfer currency internationally at the best possible rates.</p>
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.jsx:69]:
```
<p className="contactList__desc">
            All your friends and family financial details in one place. Easily transfer currency internationally at the best possible rates.
</p>
```

File: src/components/ContactContent/ContactContent.jsx [hunk 5]
```diff
@@ -3,38 +3,75 @@ import BankContact from "../BankContact/BankContact";
...
+                <Searchbar saveSearchText={setSearchText} />
+
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.jsx:71]: remove extra line

File: src/components/ContactContent/ContactContent.scss [hunk 6]
```diff
@@ -5,7 +5,13 @@
     display: flex;
     flex-direction: column;
     padding: 1rem;
-    margin-top: -4rem;
+
+    &__options {
+        display: flex;
+        justify-content: space-between;
+
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.scss:12]: remove extra line

File: src/components/ContactContent/ContactContent.scss [hunk 7]
```diff
@@ -42,8 +48,18 @@
...
+        font-size: 1rem;
+        width: 90vw;
         height: 2.5rem;
-        margin-bottom: -1.5rem;
+    }
+    .addContact{
+    font-family: $font-main;
+    width: 6rem;
+    height: 3rem;
+    // margin-left: -8rem;
```
Comment (kirsty-nology) [inline src/components/ContactContent/ContactContent.scss:59]: remove comment

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

File: src/components/ContactContent/ContactContent.scss (hunk 7 above)
Comment (blueretrograde) [reply src/components/ContactContent/ContactContent.scss]: all changes done

**Review by ChrisWakley — APPROVED** [review]
All changes actioned, merge conflict resolved.

**Review by lanniiee — APPROVED** [review]

**Review by kirsty-nology — APPROVED** [review]

**Pull request merged**